src/entities/course/ui/course-item/course-item.tsx

```diff
@@ -19,7 +19,13 @@ export const CourseItem = ({
   return (
     <section className={cx('course-item')}>
       <figure className={cx('icon-container')}>
-        <Image className={cx('course-icon')} src={iconSrc} alt={title} data-testid="course-image" />
+        <Image
+          className={cx('course-icon')}
+          src={iconSrc}
+          alt=""
```

Comment:
in my opinion this is a bad idea because `${title}-icon` is not describing what is painted on the image, plus this icons is purely decorative and doesn't contain any important context

<img width="904" alt="Screenshot 2024-10-01 at 18 11 23" src="https://github.com/user-attachments/assets/30c95cc7-6e1f-47c5-8ca6-bf5206921fd5">